packages/ens-utils/src/ensname.ts

```diff
@@ -326,6 +326,19 @@ export function getRegistrationPotential(name: ENSName): RegistrationPotential {
   }
 }
 
+/**
+ * Splits a label into its individual characters.
+ * 
+ * This function uses the spread operator to split the string into an array of its Unicode characters (UTF-32) compatible with ens-normalize.
+ * Each element in the resulting array represents a single Unicode character.
+ * 
+ * @param label
+ * @returns An array of the individual characters within `label`.
+ */
+export function splitCharacters(label: string): string[] {
+  return [...label];
+}
+
 /**
  * Calculates the number of characters in a label.
```

Comment:
Can we also update this function to make it clear that it's about any string in general and not only labels?